src/Maths/Silk.NET.Maths/Scalar.cs

```diff
@@ -248,57 +248,57 @@ public static T SquareRoot(T value)
         {
             if (typeof(T) == typeof(byte))
             {
-                return (T) (object) (MathF.Sqrt((byte) (object) value));
+                return (T) (object) (byte) (MathF.Sqrt((byte) (object) value));
```

Comment:
Don't lie about integral square root. Either this should use an integral square root path, or it should be not supported. Users will get very unexpected results doing this